### ISSUE TITLE
[1.2.12] Sunset colors aren't accurate, requires color theory

### ISSUE THREAD

**Issue by Cixon** [TathanDev/SkyAesthetics#15]
Colors are very limited, an example of this is hex color code `675A50` which would be:
```
    "sunrise_color": [
        103,
        90,
        80
    ],
```
**Bug:**
Results would look nothing like the color above. Using color theory, you could probably find it but requires more work than something as simple as direct color replacement.

**Comment by TathanDev** [comment]
I succeed to have this with the color you gave me, is what you looking for ?
![image](https://github.com/user-attachments/assets/9daeb439-56f0-4553-a5c6-c1971a8138d9)

**Comment by Cixon** [comment]
![IMG_4143](https://github.com/user-attachments/assets/ab0ad95d-da6a-460d-a2a8-b0bd8aba65c1)
This is the color, it’s not close to that in your screenshot.

**Comment by TathanDev** [comment]
Ok thanks for the color I'll check

**Comment by TathanDev** [comment]
![image](https://github.com/user-attachments/assets/0a8a60e7-8cc4-4819-aa47-1c4079285a9f)
 Something like this I suppose

**Comment by TathanDev** [comment]
![image](https://github.com/user-attachments/assets/e5f7b0b6-2ec9-4476-8984-f55dbd73d80f)
And you can modify the `sunrise_alpha_modifier` to have this (it's 1.5)

**Comment by TathanDev** [comment]
If it's what you are looking for, it will be in the version 1.3.0 that will come with some breaking changes

**Comment by Cixon** [comment]
> ![image](https://private-user-images.githubusercontent.com/67022946/389113472-0a8a60e7-8cc4-4819-aa47-1c4079285a9f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIzMTg0ODAsIm5iZiI6MTczMjMxODE4MCwicGF0aCI6Ii82NzAyMjk0Ni8zODkxMTM0NzItMGE4YTYwZTctOGNjNC00ODE5LWFhNDctMWM0MDc5Mjg1YTlmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDIzMjk0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxNmVjYjIyM2U0ZTU5ZGI2YjE1MzQ3YTMxYTlhZmJiNDliZWM1MjA5MDgzZjQ2MTA2OTQ0NDc5NzA5YTNlMjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.DUL83pgNt3EpQxhPYiMWzXOCs5kAC6zjs1VAe6NsEto)
> Something like this I suppose

This is pretty good, thanks!

**Comment by Cixon** [comment]
> If it's what you are looking for, it will be in the version 1.3.0 that will come with some breaking changes

oh? excited!

**Comment by TathanDev** [comment]
Okay so fixed in the new version  ! It's the 1.3.0 so there are small breaking changes.
You can find the updated wiki in the wiki section and all the breaking changes here : https://modrinth.com/mod/sky-aesthetics/version/M8qcK7aG !

**Comment by Cixon** [comment]
thank you for everything. seriously!
